.travis.yml

```diff
@@ -21,8 +21,6 @@ env:
     - INSTALL_PREFIX="$HOME/nvim-install"
     # Log directory for Clang sanitizers and Valgrind.
     - LOG_DIR="$BUILD_DIR/log"
-    # Nvim log file.
-    - NVIM_LOG_FILE="$BUILD_DIR/.nvimlog"
```

Comment:
Seems to be cleaner to keep it here, no?
Is `$BUILD_DIR` already setup here (defined above)?  (but it seems to work for `$LOG_DIR`?!